app/models/form/response.js

```diff
@@ -32,6 +32,11 @@ export default class Response extends DirtySaveModel {
             await this.closedQuestionAnswers
           ).map((closedQuestionAnswer) => closedQuestionAnswer.option)
         );
+        answers.sort((answer1, answer2) =>
+          answer1.option && answer2.option
+            ? answer1.option.get('position') - answer2.option.get('position')
+            : 0
+        );
```

Comment:
```suggestion
        answers.sortBy('position');
```

Reply:
This change does not show the correct ordering in the sign-up table rows when I try it

Reply:
yeah I don't know why I thought this would work, and I'm unsure why I didn't see any change in the ordering when testing it.